### DISCUSSION TITLE
8.6.2 & S3

### DISCUSSION THREAD

**Discussion by jhines0908** [iterate-ch/cyberduck#14971] [Q&A] [answered]
Anyone having issues with S3 running MacOS (app store) 8.6.2 and MacOS 13.5? I keep getting the dreaded "can't read bucket location" error, but reverting back to 8.5.9 works like a champ.

**Comment by TomG-LifeWay** [comment]
Same here - doesn't seem to recognize the creds when using the AWS S3 CLI

**Comment by dkocher** [comment]
Do you use AWS SSO as in #14970?

**Comment by TomG-LifeWay** [comment]
Yes. Revert back to 8.6.0 and connects without issue.

**Comment by dkocher** [comment, marked as answer]
We are working on a fix in #14970.
